core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -5450,6 +5450,9 @@ groups:
     - openshift-openshift-azure-infra-periodics.yaml
     - openshift-release-infra-periodics.yaml
     - infra-periodics-origin-release-images.yaml
+  build01:
+    paths:
+    - .*-postsubmits.yaml$
```

Comment:
this needs to go in the build01 list above, this is a duplicate key

Reply:
Stupid me!
On it.